### PR TITLE
fix: ie10 bugs

### PR DESCRIPTION
**Pull request by jung-han**
### Please check if the PR fulfills these requirements
- [ ] It's submitted to right branch according to our branching model
- [ ] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: apps/chart/src/charts/chart.ts [hunk 1]
```diff
@@ -237,13 +237,15 @@ export default abstract class Chart<T extends Options> {
...
-      (usingContainerSize.height && !this.el.style.height) ||
-      (usingContainerSize.width && !this.el.style.width)
+      (usingContainerSize.height && !this.el.style.height.length) ||
+      (usingContainerSize.width && !this.el.style.width.length)
     ) {
       throw new Error(message.AUTO_LAYOUT_CONTAINER_SIZE_ERROR);
     }
 
-    if (isUndefined(ResizeObserver)) {
+    const isResizeObserverAPIExist = typeof ResizeObserver === 'undefined';
```
Comment (jung-han) [inline apps/chart/src/charts/chart.ts:246]: IE 비교 문제 (ResizeObserver 존재)

File: apps/chart/src/charts/chart.ts [hunk 2]
```diff
@@ -576,7 +578,7 @@ export default abstract class Chart<T extends Options> {
    */
   public resize = (size: Partial<Size>) => {
     this.resetSeries();
-    this.store.dispatch('updateOptions', { options: { chart: { ...size } } });
+    this.dispatchOptionsEvent('updateOptions', { chart: { ...size } });
```
Comment (jung-han) [inline apps/chart/src/charts/chart.ts:581]: containerEL이 넘어가지 않던 부분 수정

File: apps/chart/src/charts/chart.ts [hunk 3]
```diff
@@ -610,16 +612,16 @@ export default abstract class Chart<T extends Options> {
     const isAutoWidth = isAutoValue(width);
     const isAutoHeight = isAutoValue(height);
 
+    this.store.dispatch(
+      'setUsingContainerSize',
+      getUsingContainerSize(eventName, usingContainerSize, width, height)
+    );
+
```
Comment (jung-han) [inline apps/chart/src/charts/chart.ts:619]: usingContainerSize를 기준으로 이벤트를 걸지 말지 판단하고 있는데 업데이트가 추가된뒤에 이뤄져 위치 이동

File: apps/chart/src/component/exportMenu.ts [hunk 4]
```diff
@@ -156,9 +159,9 @@ export default class ExportMenu extends Component {
 
   applyPanelWrapperStyle() {
     const exportMenu = this.exportMenuEl.querySelector('.toastui-chart-export-menu')!;
-    const { top, left, width } = this.chartEl.getBoundingClientRect();
+    const { top, left } = this.chartEl.getBoundingClientRect();
     const { scrollX, scrollY } = getScrollPosition();
-    const x = left + width - EXPORT_MENU_WIDTH - padding.X + scrollX;
+    const x = left + this.chartWidth - EXPORT_MENU_WIDTH - padding.X + scrollX;
```
Comment (jung-han) [inline apps/chart/src/component/exportMenu.ts:164]: container로 export 메뉴를 추가하면 컨테이너보다 작은 경우  위치가 밖으로 나가는 문제

File: apps/chart/src/store/root.ts [hunk 5]
```diff
@@ -27,6 +27,7 @@ const root: StoreModule = {
     setChartSize({ state }, size: Size) {
       state.chart.width = size.width;
       state.chart.height = size.height;
+      this.notify(state, 'chart');
```
Comment (jung-han) [inline apps/chart/src/store/root.ts:30]: legend 너비 계산을 위한 수정(auto)

**Review by jungeun-cho — APPROVED** [review]
수고하셨습니다.

**Pull request merged**